guides/ansible_tower/2-config/README.md

```diff
@@ -0,0 +1,172 @@
+# Exercise 2 - Configuring Ansible Tower
+
+In this exercise, we are going to configure Tower so that we can run a playbook.
+
+## The Tower UI
+
+There are a number of contructs in the Ansible Tower UI that enable multi-tenancy, notifications, scheduling, etc. However, we are only going to focus on a few of the key contructs that are required for this workshop today.
```

Comment:
@dfederlein I don't think constructs is our standardized language typically. What does your team use? Concepts?

Reply:
In the documentation when we speak about playbooks and tags, it is called "concepts". See for example here: http://docs.ansible.com/ansible-tower/latest/html/userguide/